src/aics-image-viewer/components/App/index.tsx

```diff
@@ -952,202 +540,277 @@ export default class App extends React.Component<AppProps, AppState> {
+            resetCamera={resetCamera}
+            downloadScreenshot={saveScreenshot}
+            changeViewerSetting={changeViewerSetting}
+            showControls={showControls}
+          />
+          <CellViewerCanvasWrapper
+            view3d={view3d}
+            image={image}
+            viewMode={viewerSettings.viewMode}
+            autorotate={viewerSettings.autorotate}
+            loadingImage={sendingQueryRequest}
+            numSlices={getNumberOfSlices()}
+            region={viewerSettings.region}
+            appHeight={props.appHeight}
+            showControls={showControls}
+            changeViewerSetting={changeViewerSetting}
+            onClippingPanelVisibleChange={onClippingPanelVisibleChange}
+            onClippingPanelVisibleChangeEnd={onClippingPanelVisibleChangeEnd}
           />
-        </Sider>
-        <Layout className="cell-viewer-wrapper" style={{ margin: this.props.canvasMargin }}>
-          <Content>
-            <Toolbar
-              viewMode={viewerSettings.viewMode}
-              fovDownloadHref={fovDownloadHref}
-              cellDownloadHref={cellDownloadHref}
-              autorotate={viewerSettings.autorotate}
-              imageType={viewerSettings.imageType}
-              hasParentImage={!!this.props.fovPath}
-              hasCellId={!!this.props.cellId}
-              canPathTrace={this.state.view3d ? this.state.view3d.hasWebGL2() : false}
-              showAxes={viewerSettings.showAxes}
-              showBoundingBox={viewerSettings.showBoundingBox}
-              renderMode={viewerSettings.renderMode}
-              onViewModeChange={this.onViewModeChange}
-              onResetCamera={this.onResetCamera}
-              onAutorotateChange={this.onAutorotateChange}
-              onSwitchFovCell={this.onSwitchFovCell}
-              onChangeRenderingAlgorithm={this.onChangeRenderingAlgorithm}
-              changeAxisShowing={this.changeAxisShowing}
-              changeBoundingBoxShowing={this.changeBoundingBoxShowing}
-              downloadScreenshot={this.saveScreenshot}
-              showControls={showControls}
-            />
-            <CellViewerCanvasWrapper
-              view3d={this.state.view3d}
-              image={this.state.image}
-              setAxisClip={this.setImageAxisClip}
-              viewMode={viewerSettings.viewMode}
-              autorotate={viewerSettings.autorotate}
-              loadingImage={this.state.sendingQueryRequest}
-              numSlices={this.getNumberOfSlices()}
-              region={viewerSettings.region}
-              appHeight={this.props.appHeight}
-              showControls={showControls}
-              onClippingPanelVisibleChange={this.onClippingPanelVisibleChange}
-              onClippingPanelVisibleChangeEnd={this.onClippingPanelVisibleChangeEnd}
-            />
-          </Content>
-        </Layout>
+        </Content>
       </Layout>
-    );
-  }
-}
+    </Layout>
+  );
+};
+
+export default App;
```

Comment:
300 fewer lines of code. 👍 
I still think about having a completely separate view3d interface class that does the translation between a bunch of our react state and the volume-viewer's api.  To try to keep the react coupling looser and put that responsibility into its own code module.  I have no idea if that would really make things better. This is looking good.